CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+### Added
+- Added support for JSON output to `ic-wasm info`.
```

Comment:
Should I bump the version number in this PR?

Reply:
Yeah, feel free to bump the version and merge.

Reply:
Thank you for the review.  Bumped the version and merging...